### PR TITLE
email notification custom action to be used with workflows

### PR DESCRIPTION
**Pull request by ShristyBhati21**

### CODE REVIEW

File: action.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# action.yml
```
Comment (ryand-byzgen) [inline action.yml]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -1 +1,20 @@
-# email-notification-action
\ No newline at end of file
+# The email-notification-action
+
+This project is designed to create custom email notifications via github actions.
+
+To add this email-notification-action to your builds, just add the following in your worflow file :
```
Comment (ryand-byzgen) [inline README.md]:
```suggestion
To add this email-notification-action to your builds, just add the following in your workflow file:
```

**Review by ryand-byzgen — APPROVED** [review]

File: files/message_template.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <title>Bootstrap Example</title>
```
Comment (robevansuk) [inline files/message_template.html]: probably want to update the title here

**Review by robevansuk — APPROVED** [review]

File: entrypoint.sh [hunk 4]
```diff
@@ -0,0 +1,25 @@
+#!/bin/sh -l
+
```
Comment (ryand-byzgen) [inline entrypoint.sh:6]:
```suggestion
#
# Short description of what this script does
#
```

File: files/message_template.html [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+    <div class="row">
+        <div class="col-sm-5">
+            <h3> <b> Result :</b> </h3>
+            <p><a href='$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID'> Commit Action </a> <br> Branch: <a href='$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/tree/$GITHUB_REF_NAME'>$GITHUB_REF_NAME</a> <br> Build: #$GITHUB_RUN_NUMBER <br> By: $GITHUB_ACTOR <br> <a href='$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/commit/$GITHUB_SHA'>commit</a></p>
+        </div>
+        <div class="col-sm-1">
+            <img src=$INPUT_DISPLAY_IMAGE_URL width="104" height="142">
+        </div>
+    </div>
+</div>
```
Comment (ryand-byzgen) [inline files/message_template.html]: Indent this part

**Review by ryand-byzgen — APPROVED** [review]

**Pull request merged**